bin/edgex-mongo-launch.sh

```diff
@@ -8,22 +8,28 @@
 #Exit the script immediately if a command exits with a non-zero status
 set -e
 
-###
-# Run MongoDB bind to localhost.
-###
-mongod &
-
-###
-# Run Edgex-Mongo Go Application and initiate the database
-###
-cd cmd/
-./edgex-mongo --profile=docker --confdir=res
-
-
-###
-# Restart Edgex-Mongo with enabled authentication and bind_ip_all
-###
-mongod --shutdown
+#hiden file used for determine if the database was already initialized
+DB_INITIALIZATION_FLAG=/tmp/edgex/secrets/edgex-mongo/.mongo-database-setup-done
```

Comment:
The secrets volumes /tmp/edgex/secrets/* are generally read-only to consuming containers.  Will check this out during validation and suggest an alternative path if my suspicion is true.  Please hold off merging until I can validate.

Reply:
O, you are right !! I forgot.. have been done some changes in docker-compose file from ro -> rw.
I will fix it tomorrow.

Reply:
Don't do that!  Just change it to `/tmp/mongo-database-setup-done` or something.  It is read-only in order to enforce least privilege.

Reply:
Yes, yes..I meant to change the directory ...not to use `/tmp` directory